vaas-app/setup.py

```diff
@@ -73,40 +69,19 @@ def run(self):
         org_egg_info.run(self)
 
 
-def extract_requirement(requirement):
-    req = None
-    link = None
-    if hasattr(requirement, 'req'):
-        req = str(getattr(requirement, 'req'))
-        if hasattr(requirement, 'link'):
-            link = str(getattr(requirement, 'link'))
-    elif hasattr(requirement, 'requirement'):
-        from pip._internal.req.constructors import parse_req_from_line
-        parts = parse_req_from_line(requirement.requirement, requirement.line_source)
-        req = requirement.requirement
-        if parts.requirement.url:
-            link = str(parts.requirement.url)
-    assert req is not None, "Unknown requirement, cannot find properties req or requirement {}".format(requirement.__dict__)
-    return req, link
-
-
 base_requirements = []
 test_requirements = []
-dependency_links = []
 
-for index, requirement in enumerate(parse_requirements('{}/requirements/base.txt'.format(current_dir), session=False)):
-    req, dependency_link = extract_requirement(requirement)
-    base_requirements.append(req)
-    if dependency_link:
-        dependency_links.append(dependency_link)
+with open('{}/requirements/base.txt'.format(current_dir)) as f:
+    for line in f.read().splitlines():
+        if not line.lstrip().startswith('-'):
```

Comment:
You ignore `-` sign in purpose of comment, not a `#` ?

Reply:
We have to ignore -r in requirements because it can be only used in console tools like pip install. I also added # to be ignored.